apps/web-desmos/src/screens/proposal_details/components/overview/index.tsx

```diff
@@ -47,45 +51,49 @@ const Overview: FC<{ className?: string; overview: OverviewType }> = ({ classNam
 
   const getExtraDetails = useCallback(() => {
     let extraDetails = null;
-    if (type === 'parameterChangeProposal') {
-      extraDetails = (
-        <>
-          <Typography variant="body1" className="label">
-            {t('changes')}
-          </Typography>
-          <ParamsChange changes={R.pathOr([], ['changes'], overview.content)} />
-        </>
-      );
-    } else if (type === 'softwareUpgradeProposal') {
-      extraDetails = (
-        <>
-          <Typography variant="body1" className="label">
-            {t('plan')}
-          </Typography>
-          <SoftwareUpgrade
-            height={R.pathOr('0', ['plan', 'height'], overview.content)}
-            info={R.pathOr('', ['plan', 'info'], overview.content)}
-            name={R.pathOr('', ['plan', 'name'], overview.content)}
-          />
-        </>
-      );
-    } else if (type === 'communityPoolSpendProposal') {
-      extraDetails = (
-        <>
-          <Typography variant="body1" className="label">
-            {t('content')}
-          </Typography>
-          <CommunityPoolSpend
-            recipient={overview?.content?.recipient}
-            recipientMoniker={recipientMoniker}
-            amountRequested={parsedAmountRequested}
-          />
-        </>
-      );
-    }
+    types.forEach((type: string) => {
+      if (type === 'parameterChangeProposal') {
+        extraDetails = (
```

Comment:
So this loop will set extraDetails depending of the last value of the types array, not sure of logic but just to confirm that this is expected